indico/modules/events/themes.yaml

```diff
@@ -0,0 +1,248 @@
+defaults:
+    conference: static
+    simple_event: lecture
+    lecture: lecture
+    meeting: standard
+definitions:
+  # Lectures - base
+  event:
+    event_types: [simple_event]
```

Comment:
could use `event.type` instead of `conf.getType()` in the places where you access this - that way you could use `lecture` instead of `simple_event`